src/narrow-screen/screens/MenuScreen.js

```diff
@@ -191,28 +197,33 @@ export default () => {
   // Must be triggered before render
   useLayoutEffect(() => {
     const newCategoryToRef = {};
+    const newCategoryToTabRef = {};
     context.menu.categories.forEach((c) => {
-      const categoryRef = React.createRef();
-      newCategoryToRef[c.id] = categoryRef;
+      newCategoryToRef[c.id] = React.createRef();
+      newCategoryToTabRef[c.id] = React.createRef();
     });
-
-    setContainerRef(React.createRef)
+    setTabBarRef(React.createRef());
     setCategoryToRef(newCategoryToRef);
+    setCategoryToTabRef(newCategoryToTabRef);
     setActiveCategoryId(context.menu.categories[0]?.id);
   }, [context.menu]);
 
+  useLayoutEffect(() => {
+    window.scrollTo({ top: 0 });
+  }, [context.selectedMenuIndex]);
+
+  // Triggered by mouse wheel or panning (swiping up/down) on mobile
+  function onWheel() {
```

Comment:
Are we using this function?

Reply:
No - also as we discussed we don't need to worry about hiding part of nav bar now, so I'm just gonna remove this callback.